addons/web/static/src/webclient/webclient.js

```diff
@@ -99,7 +99,7 @@ export class WebClient extends Component {
         // we let the browser do the default behavior and
```

Comment:
About the commit message:
- You wrote "clic" instead of "click" multiple times.
- Missing task-id
- I think the commit message is not clear enough about why this is in fact an important UX issue
- I think the commit message is not clear enough about the fact that there were actually two different (albeit related) issues at play here that prevented the feature:
1. We knew clicking on a link in a contentEditable wouldn't open the link but listening for a click didn't work because of 2.
2. The click was not received at the editor level at all because of the `stopPropagation` hook.

I suggest the following rewrite of the commit title and message:
```
[IMP] web_editor: Ctrl+Click open links in new tab

By default, clicking on links in a contentEditable context does not open the
link because it would make it hard to edit it otherwise.

This is a problem now that the views are always in edit mode because it means
the user cannot quickly and easily click on links to open them.

This commit aims to solve this conundrum by finding a middle-ground between
these two use cases, that is reintroducing the oh-so loved Ctrl+Click behavior
that was commonly used in previous versions.

This required two changes:
1. Explicitely implementing the behavior of opening the link in a new tab in
   that case since it is not the native behavior in a contentEditable context.
2. Giving special treatment to links inside a contentEditable context in the
   `globalClick` listener introduced at [1] such that the event can actually
   reach the custom handler in the editor.

task-id 3188438

[1]: https://github.com/odoo/odoo/commit/47acba560a8954f10bfa97b28daf90047005fdf1#diff-ce4923ebad38db28d4fc17267c93d147c4404feefd42eb140ef990bddbe1a9ffR93
```

Reply:
I really like your rewrite, thank you for. I'll try to improve my style of writing to be more concise, informative and readable.